modules/extensions/scripts/install_kubectx.template.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Copyright 2022 Oracle Corporation and/or affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
+
+curl -sSfL https://github.com/ahmetb/kubectx/releases/download/v0.9.4/kubectx_v0.9.4_linux_x86_64.tar.gz -o kubectx_v0.9.4_linux_x86_64.tar.gz
+curl -sSfL https://github.com/ahmetb/kubectx/releases/download/v0.9.4/kubens_v0.9.4_linux_x86_64.tar.gz -o kubens_v0.9.4_linux_x86_64.tar.gz
+
+tar zxf  kubectx_v0.9.4_linux_x86_64.tar.gz
+tar zxf kubens_v0.9.4_linux_x86_64.tar.gz
+rm -f  kubectx_v0.9.4_linux_x86_64.tar.gz
+rm -f kubens_v0.9.4_linux_x86_64.tar.gz
+rm -f LICENSE
+
+sudo mv kubectx /usr/local/bin
+sudo mv kubens /usr/local/bin
```

Comment:
Instead of moving, create symbolic links to `/usr/bin`:

```bash
ln -s /usr/bin/kubectx /usr/local/kubectx/kubectx
ln -s /usr/bin/kubens /usr/local/kubens/kubens
```

Reply:
Ok but I think it should be 

```
sudo ln -s /usr/local/kubectx/kubectx /usr/bin/kubectx

sudo ln -s /usr/local/kubens/kubens /usr/bin/kubens 
```

Reply:
You're right. I always get symlinks the wrong way around.